src/OrchardCore.Modules/OrchardCore.Settings/Startup.cs

```diff
@@ -40,6 +40,7 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<ILiquidTemplateEventHandler, SiteLiquidTemplateEventHandler>();
 
             services.AddScoped<ITimeZoneSelector, DefaultTimeZoneSelector>();
+            services.AddScoped<ICalendarSelector, DefaultCalednarSelector>();
```

Comment:
DefaultCalednarSelector => DefaultCalendarSelector

Reply:
Nice catch .. it's my fault 😊